### PR TITLE
Update session account profile

### LINKED ISSUE #60: implement PATCH /session/account/profile
The `/session/account/profile` route is for accessing and changing non-sensitive user data

---

- [x] claim this issue (@antoinelyset)
- [ ] start a pull request
- [ ] write tests for `PATCH /session/account/profile` in [tests/integration/routes/account-test.js](https://github.com/hoodiehq/hoodie-server-account/blob/master/tests/integration/routes/account-test.js)
- [ ] implement `PATCH /session/account/profile`
- [ ] update README
- [ ] review & merge

Ping me in the [Hoodie Chat](http://hood.ie/chat/) or on [Twitter](https://twitter.com/gr2m/) if you have any questions

### PR DESCRIPTION
**Pull request by antoinelyset**
Hi,

I will need some help on the tests if you've got the time. Right now I've got an error ```Missing JSON list of 'docs'```. I think it comes front the fact that I'm not able to properly mock the request to update the account. (And I think, I need to mock 2 times the GET with ```mockUserFound```). Am I right ?

Thanks for your work too ! What a nice community ! :heart_eyes_cat: 

fixes #60

### CODE REVIEW

**Comment by gr2m** [comment]
This "Missing JSON list of 'docs'" error is really annoying, not very helpful. I filed an issue for it here:
https://github.com/pouchdb/pouchdb/issues/4658

Yes, you are right, the `.get('/_users/org.couchdb.user%3Apat-doe')` mock needs to be mocked twice for updates at this point. And you also need to mock the update request, which for PouchDB is a `.post('/_users/_bulk_docs')`. See `accounts-test.js` for comparision

**Comment by antoinelyset** [comment]
That wasn't the easiest tests I've ever written, but this is done.
I've a bit diverged from the [original spec](http://docs.accountjsonapi.apiary.io/#reference/current-user/profile/update) to be consistent with ``` PATCH /accounts/{id}```. And of course everything is modifiable.

File: tests/integration/routes/profile-test.js [hunk 1]
```diff
@@ -98,4 +112,71 @@ getServer(function (error, server) {
 
     group.end()
   })
+
+  test.only('PATCH /session/account/profile', function (group) {
```
Comment (gr2m) [inline tests/integration/routes/profile-test.js]: interestingly, `test.only` causes this test to fail but it doesn’t appear on `npm test`, only on `node tests/integration/routes/profile-test.js` ... kinda tricky :) 

But in your case the tests look good, just remove the `.only()` part

Comment (antoinelyset) [inline tests/integration/routes/profile-test.js]: Haha, I forget to remove this ! Thanks :)

**Comment by antoinelyset** [comment]
Should be good :)

**Comment by gr2m** [comment]
woop :+1: great work @antoinelyset, thanks

**Pull request merged**